src/stores/useSuosikitStore/index.ts

```diff
@@ -90,22 +90,31 @@ export const useSuosikitStore = create<FavoritesState>()((set, get) => ({
       return;
     }
 
+    const hasTyomahdollisuus = suosikit.findIndex((s) => s.tyyppi === 'TYOMAHDOLLISUUS') > -1;
+    const hasKoulutusMahdollisuus = suosikit.findIndex((s) => s.tyyppi === 'KOULUTUSMAHDOLLISUUS') > -1;
+
     const paginated = paginate(suosikit, page, pageSize);
```

Comment:
I fixed an issue (PR #246 ) with the paging of favorites so there is some changes when rebasing this. Remembered your PR so I took some logic from this part to be used there already.